l10n_it_delivery_note_inter_warehouse/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* PyTech SRL <info@pytech.it>
```

Comment:
chore: (just leaving a comment here because I can't comment on empty files) please fill at least DESCRIPTION; remove files that you don't use or need.

Reply:
This is still left to do. Also, as discussed in other PRs, please add people and not just companies.